net-im/slack-bin/slack-bin-3.0.5.ebuild

```diff
@@ -15,7 +15,7 @@ SRC_URI="https://downloads.slack-edge.com/linux_releases/${MY_PN}-desktop-${PV}-
 LICENSE="all-rights-reserved"
```

Comment:
For the future: use one of 'Bug' or 'Closes' in the commit message, not both. I'll fix it for you.

Reply:
Ok.